stdlib/2and3/grp.pyi

```diff
@@ -0,0 +1,10 @@
+from typing import List, NamedTuple
+
+struct_group = NamedTuple("struct_group", [("gr_name", str),
+                                           ("gr_passwd", str),
+                                           ("gr_gid", int),
+                                           ("gr_mem", List[str])])
```

Comment:
I'm looking into implementation of modules:

- https://github.com/python/cpython/blob/2.7/Modules/grpmodule.c#L35-L83
- https://github.com/python/cpython/blob/3.6/Modules/grpmodule.c#L40-L83

`gr_name`: This can't be `None`, so type annotation should be `str`
`gr_passwd`: This can be `None`, so type annotation should be `Optional[str]`
`gr_name`: This can't be `None`, so type annotation should be `str`
`gr_mem`: This can't be `None`, so type annotation should be `str`

It seems that we can use the same type for both Python 2.7 and Python 3.

(I'm not used to reading modules written in C, so please feel free to point out my misunderstandings)